war/src/main/java/org/exoplatform/addons/es/EmbeddedESStartupServlet.java

```diff
@@ -27,11 +28,20 @@
 
   private static final Log LOG = ExoLogger.getLogger(EmbeddedESStartupServlet.class);
 
+  public static final String ES_EMBEDDED_ENABLED_PROPERTY_NAME = "exo.es.embedded.enabled";
```

Comment:
I was not sure since it is related to an addon, but since it will be packaged by default in PLF, I think you are right. I will do this as part of the issue related to the integration of the ES Embedded in PLF.
Thanks